src/lib/utils/countries.ts

```diff
@@ -68,6 +68,7 @@ export const countriesList = [
   { alpha2: "ER", name: "Eritrea" },
   { alpha2: "ES", name: "Spain" },
   { alpha2: "ET", name: "Ethiopia" },
+  { alpha2: "EU", name: "European Union" },
```

Comment:
L'unione europa non è uno stato. Se hai bisogno di metterla in pandora implementati la tua countries autocomplete nel progetto.